src/test/java/StadAlexTest.java

```diff
@@ -0,0 +1,49 @@
+    import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+    public class StadAlexTest extends BaseTest {
+
+        @Test
+        public void testH2TagText_WhenSearchingCityCountry() throws InterruptedException {
+
+
+            String url = "https://openweathermap.org/";
+            String cityName = "Paris";
+            String expectedResult = "Paris, FR";
+
+            getDriver().get(url);
+            Thread.sleep(5000);
+
+            WebElement searchCityField = getDriver().findElement(
+                    By.xpath("//div[@id = 'weather-widget']//input[@placeholder = 'Search city']")
+            );
+            searchCityField.click();
+            searchCityField.sendKeys(cityName);
+            WebElement searchButton = getDriver().findElement(
+                    By.xpath("//div[@id='weather-widget']//button[@type='submit']")
+            );
+            searchButton.click();
+            Thread.sleep(2000);
+
+            WebElement parisFRChoiceInDropDownMenu = getDriver().findElement(
+                    By.xpath("//ul[@class=\"search-dropdown-menu\"]//li//span[text() = 'Paris, FR ']")
+            );
+            parisFRChoiceInDropDownMenu.click();
+            Thread.sleep(2000);
+
+            WebElement h2CityCountryHeader = getDriver().findElement(
+                    By.xpath("//div[@id = 'weather-widget']//h2")
+            );
+            Thread.sleep(2000);
+
+            String actualResult = h2CityCountryHeader.getText();
+            Assert.assertEquals(actualResult, expectedResult);
+
+//        Thread.sleep(2000);
+
+            getDriver().quit();
+        }
+    }
```

Comment:
```suggestion
    }
```
После последней строки не забывайте нажимать Enter.

И почему класс сдвинут вправо?